Add documentation for LED Infrared integration (#46600)
home-assistant/home-assistant.github.io: source/_integrations/led_infrared.markdown

---
title: LED Infrared
description: Integration to control LED infrared lights with Home Assistant.
ha_category:
  - Light
ha_release: 2026.8
ha_iot_class: Assumed State
ha_codeowners:
  - '@tr4nt0r'
ha_domain: led_infrared
ha_config_flow: true
ha_platforms:
  - light
ha_integration_type: device
---

> The **LED Infrared** {% term integration %} lets you control lights using any infrared transmitter previously configured in Home Assistant.

Reviewer: ```suggestion
The **LED Infrared** {% term integration %} lets you control lights with any infrared emitter that has been previously configured in Home Assistant.
```
From what I understand, we use the term "transmitter" with RF devices, and "emitter" with infrared.

Because the integration communicates over infrared, it operates in a one-way, fire-and-forget fashion: commands are sent to the light, but there is no feedback channel to confirm the current state of the light. The integration therefore uses assumed states.

## Supported devices

This integration supports many branded and unbranded LED bulbs, lamps, and LED strip controllers that use generic infrared remotes. If your device uses one of the remote controls listed below, it may be compatible with this integration.

{% details "24-key remote" %}

![24-key remote](/images/integrations/led_infrared/24-key.png)

{% enddetails %}

{% details "13-key remote" %}

![13-key remote](/images/integrations/led_infrared/13-key.png)

{% enddetails %}

## Prerequisites

Before setting up the LED Infrared integration, you need a working infrared transmitter set up in Home Assistant that exposes an [Infrared](/integrations/infrared/) entity. For example, you can use an ESPHome device with an IR LED pointed at your lights.

{% include integrations/config_flow.md %}

> 

Reviewer: ```suggestion

Select the device type that matches your remote control, then select at least one infrared emitter or receiver. Selecting only the device type is not enough to complete the setup.

```
Not sure. Do I need at least one?
